cmd/commands/flatten_rbd_pvc.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TrashLsOutput struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+var namespace string
+var allowInUse bool
+
+// FlattenRBDPVCCmd represents the rook commands
+var FlattenRBDPVCCmd = &cobra.Command{
+	Use:   "flatten-rbd-pvc",
+	Short: "Flatten the RBD image corresponding to the target RBD PVC",
+	Args:  cobra.ExactArgs(1),
```

Comment:
How about adding `Long` description of what it does internally for `temp` image as well?